### ISSUE TITLE
Enhancement: Get back to the previous fit

### ISSUE THREAD

**Issue by Hexanders** [Julian-Hochhaus/LG4X-V2#13]
It would be good if you could add a button to set the fit parameters to the previous parameters. If the fit runs into the nervana.

**Comment by Julian-Hochhaus** [comment]
I thought about this too, maybe it is the easiest to have a log file where the parameters are saved after each fit process. Or maybe just have an array, where something like only the last 10 or so fitting parameter sets are saved to.

**Comment by Hexanders** [comment]
Or may be it is time to introduce a parameterset class? Just simple one with some attributes as parameters/type an so one. In this way you are more flexible. So one can step by step translate everything in to this class, starting just with the „undo“ functionality for the fit.

**Comment by Julian-Hochhaus** [comment]
Indeed, it would make several things easier and would massively improve the code. But that will need major changes in the code. For backward compatibility, we will need some type of conversion between the old .dat format and some kind of parameter class. If you want to start working on that topic, feel free to do so.

I will tackle this topic maybe the earliest in autumn. :)

**Comment by Julian-Hochhaus** [comment]
Getting back to previous fit when the model was changed in between does not work currently

**Comment by Julian-Hochhaus** [comment]
- Add Redo Button
